Frameworks/UIKit/UIView.mm

```diff
@@ -257,10 +265,15 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
     }
 
     // Get the point value in this view's parent UIWindow's coordinates
-    WUIPointerPoint* pointerPoint = [pointerEventArgs getCurrentPoint:(owningWindow ? owningWindow.layer._xamlElement : nil)];
+    UIElement owningWindowElement = nullptr;
```

Comment:
It is extremely necessary because yes, the default constructor actually creates an object.

... usually. I'm not sure that UIElement is actually activatable, so I'm not sure what happens there.

In any case, you usually want to say "winrt::SomeType myObj = nullptr;" rather than "winrt::SomeType myObj;"

---
In reply to: [109245698](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109245698) [](ancestors = 109245698)